server/boot/randomAPIs.js

```diff
@@ -171,26 +171,27 @@ module.exports = function(app) {
 
   function unsubscribeAll(req, res, next) {
     req.checkParams('email', 'Must send a valid email').isEmail();
-    return User.findOne({ where: { email: req.params.email } }, (err, user) => {
+    var query = { email: req.params.email };
+    var params = {
+      sendQuincyEmail: false,
+      sendMonthlyEmail: false,
+      sendNotificationEmail: false
+    };
+    var options = { multi: true };
+    return User.update(query, params, options, (err, updates) => {
```

Comment:
For the `update()` function, I found the documentation in this link: https://mongodb.github.io/node-mongodb-native/api-generated/collection.html. I've never worked with loopback, but it seems to be a middleware between the actual backend and the frontend. Is that correct?